src/components/PredictionGame.tsx

```
@@ -177,306 +219,372 @@ interface AssignmentPredictionProps {
+          {/* Expanded Header */}
+          <div className="flex items-start justify-between">
+            <h3 className="border-l-4 border-red-500 pl-2 text-xl font-bold text-white">
+              {assignment.movie
+                ? highlightText(assignment.movie.title, searchQuery)
+                : "Unknown Movie"}
+            </h3>
+            {hasAllGuesses && (
+              <button
+                onClick={() => setUserExpanded(false)}
+                className="rounded p-1 text-gray-400 transition-colors hover:bg-gray-800 hover:text-white"
+              >
```

Biome (2.4.7)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)